windows/win_hostname.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+DOCUMENTATION = '''
+---
+module: win_hostname
+version_added: "2.0"
+short_description: Changes the hostname of a windows machine
+description:
+	- Changes the Hostname of a windows machine
+options:
+  name:
+    description:
+	  - New Hostname of the windows machine 
+	  required: true
+	  default: null
+	  aliases: []
+  restart:
+    description:
+	  - Restart after name change
```

Comment:
Maybe worth adding that machine won't be joined to the domain until reboot has happened (unlike changing the hostname)?